composer.json

```diff
@@ -36,6 +36,8 @@
         "otgs/unit-tests-framework": "~1.2.0",
         "wpml/page-builders": "dev-develop",
         "wpml/fp": "^0.1.2",
-        "wpml/collect": "dev-wpml-collect-rename"
+        "wpml/collect": "dev-wpml-collect-rename",
+        "composer/composer": "^1.10",
+        "wpml/wp": "dev-master"
```

Comment:
@strategio For these libraries we are moving away from the master branch. We use the develop branch and tag at various points for a release. Can you add your wp change to develop branch and use that.